mostlyai/engine/domain.py

```diff
@@ -153,25 +153,37 @@ class DifferentialPrivacyConfig(CustomBaseModel):
         ge=0.0,
         le=10000.0,
     )
+    delta: float = Field(
+        default=1e-5,
+        description="The delta value for differential privacy. It is the probability of the privacy guarantee not "
+        "holding. The smaller the delta, the more confident you can be that the privacy guarantee holds. This delta "
+        "will be equally distributed between the analysis and the training phase.",
+        ge=0.0,
+        le=1.0,
+    )
     noise_multiplier: float = Field(
         default=1.5,
         alias="noiseMultiplier",
-        description="The ratio of the standard deviation of the Gaussian noise to the L2-sensitivity of the function "
-        "to which the noise is added (How much noise to add).\n",
+        description="Determines how much noise while training the model with differential privacy. This is the ratio of "
+        "the standard deviation of the Gaussian noise to the L2-sensitivity of the function to which the noise is added.",
         ge=0.0,
         le=10000.0,
     )
     max_grad_norm: float = Field(
         default=1.0,
         alias="maxGradNorm",
-        description="The maximum norm of the per-sample gradients for training the model with differential privacy.\n",
+        description="Determines the maximum impact of a single sample on updating the model weights during training with "
+        "differential privacy. This is the maximum norm of the per-sample gradients.",
         ge=0.0,
         le=10000.0,
     )
-    delta: float = Field(
-        default=1e-5,
-        description="The delta value for differential privacy. It is the probability of the privacy guarantee not "
-        "holding. The smaller the delta, the more confident you can be that the privacy guarantee holds.",
+    value_protection_epsilon: float | None = Field(
+        default=1.0,
+        alias="valueProtectionEpsilon",
+        description="The DP epsilon of the privacy budget for determining the value ranges, which are gathered prior to "
+        "the model training during the analysis step. Only applicable if value protection is True. Privacy budget will be "
+        "equally distributed between the columns. For categorical we calculate noisy histograms and use a noisy threshold. "
+        "For numeric we calculate bounds and then the bounded 1st and 99th percentiles.",
```

Comment:
shall we align code with that? because otherwise, [it's not entirely true](https://github.com/mostly-ai/mostlyai-engine/pull/103/files#diff-f5acfbb7c4e2eb8ad47701dd83e82cb7f6b46b232a9ec9c76131bae3a2ac37c4R71)